master/buildbot/util/identifiers.py

```diff
@@ -22,7 +22,7 @@
 
 from buildbot import util
 
-ident_re = re.compile('^[a-zA-Z_-][a-zA-Z0-9_-]*$')
+ident_re = re.compile(u'^[a-zA-Z\u00a0-\U0010ffff_-][a-zA-Z0-9\u00a0-\U0010ffff_-]*$', flags=re.UNICODE)
```

Comment:
That's a very large range. What about combining characters and other complications in Unicode? How will that affect this?

Reply:
I also see this regex is in two places. It appears to be the same. Any way to use it only once to prevent deviations of the two?